README.md

```diff
@@ -49,11 +49,11 @@ setting `CC` and `CXX` in your environment (e.g. `CC=gcc-11` and `CXX=g++-11`),
 or on Debian-based Linux systems using `update-alternatives`.
 
 You should also have Python (3.9, 3.10 or 3.11) and `pip` installed. We use
-`cmake` and the package manager `conan` to build tket. Both can be installed
-with `pip`:
+`cmake` and the package manager `conan` to build tket and pytket. The latter can
```

Comment:
Should we point out the cmake version requirement at this point?

Reply:
Will do.

Reply:
Actually it already says: "You will need at least cmake version 3.26".

Reply:
Oh yes, I missed that